### PR TITLE
add OWNERS.md

### PR DESCRIPTION
**Pull request by yangminzhu**

### CODE REVIEW

File: OWNERS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This page lists all active maintainers. This can be used for routing PRs, questions, etc. to the
+right place.
+
+# Maintainers
+
+* Kuat Yessenov ([kyessenov](https://github.com/kyessenov)) (kuat@google.com)
+* Yangmin Zhu ([yangminzhu](https://github.com/yangminzhu)) (ymzhu@google.com)
```
Comment (yangminzhu) [inline OWNERS.md]: Please let me know if I missed anyone here, thank you!

**Comment by yangminzhu** [comment]
cc @mattklein123 @htuch

File: OWNERS.md (hunk 1 above)
Comment (kyessenov) [reply OWNERS.md]: cc @davecheney : let us know if you're willing to help with maintaining this repo.

**Comment by mattklein123** [comment]
Right now the people with write access are here:
https://github.com/orgs/envoyproxy/teams/go-control-plane/members

cc @davecheney @junr03 @rshriram 

We definitely need more maintainers if @kyessenov has been doing all the work. Can we reconcile whether the above three still want to be maintainers? If so please add them to OWNERS.md? If not we can remove their write access?

**Comment by kyessenov** [comment]
Merging to kick-start the initial list. @davecheney @rshriram let us know if you would like to be added later.

**Pull request merged**